ee/src/main/java/io/supertokens/ee/EEFeatureFlag.java

```diff
@@ -225,37 +194,28 @@ private boolean isEnterpriseThirdPartyId(String thirdPartyId) {
         return false;
     }
 
-
     private JsonObject getMFAStats() throws StorageQueryException, TenantOrAppNotFoundException{
-        JsonObject mfaStats = new JsonObject();
-        JsonArray mfaMauArr = new JsonArray();
-
+        JsonObject result = new JsonObject();
         Storage[] storages = StorageLayer.getStoragesForApp(main, this.appIdentifier);
 
-        // TODO: Active users are present only on public tenant and MFA users may be present on different storages
```

Comment:
Double check: "TODO: Active users are present only on public tenant and MFA users may be present on different storages"

Reply:
If it's valid, also add this comment for account linking

Reply:
done